lib/otel/span-processor.js

```diff
@@ -270,6 +276,46 @@ module.exports = class NrSpanProcessor {
       [ATTR_MESSAGING_MESSAGE_CONVERSATION_ID]: (value) => segment.addAttribute('correlation_id', value),
       [ATTR_MESSAGING_RABBITMQ_DESTINATION_ROUTING_KEY]: (value) => segment.addAttribute('routing_key', value)
     }
+
     this.#reconciler.reconcile({ segment, otelSpan: span, mapper })
   }
+
+  addAWSLinkingAttributes({ segment, span }) {
+    // Applicable to all AWS services
+    const accountId = this.agent?.config?.cloud?.aws?.account_id
```

Comment:
this is going to add the account_id from config to all segments, non aws. we only need to add this and `cloud.region` to SQS related segments. i'd move this and cloud.region logic within the check on rpc.service === 'amazonsqs`